src/Events/Playlist/index.js

```diff
@@ -1,14 +1,29 @@
-import { SOCKET_EVENTS } from '../../Constants';
+import { SOCKET_EVENTS, PLAYLIST_STATUS } from '../../Constants';
 import RoomService from '../../Services/Room';
 
-export const addVideoToPLaylist = async ({ id, username, link }, { socket, io }) => {
-  await RoomService.addVideoToPLaylist({ id, username, link });
-  io.to(id).emit(SOCKET_EVENTS.PLAYLIST_UPDATED, { id, username, link });
+export const updatePlaylist = async (
+  { id, videoId, username, link, status }, { socket, io }
+) => {
+  switch (status) {
+    case PLAYLIST_STATUS.ADD:
+      await RoomService.addVideoToPLaylist({ id, username, link });
+      break;
+    case PLAYLIST_STATUS.MOVE_DOWN:
+      await RoomService.moveDown(id, videoId);
+      break;
+    case PLAYLIST_STATUS.MOVE_UP:
+      await RoomService.moveUp(id, videoId);
+      break;
+    default:
```

Comment:
```suggestion
    default:
      return;
```
eğer status belirlediğimiz şeyler arasında yoksa status olmadığı halde emit edilmemeli bu yüzden return yazabiliriz.